uiplib/gui/settingsTab.py

```diff
@@ -146,7 +146,20 @@ def handle_settings(appObj):
         return
     update_settings(appObj.new_settings)
 
-
+def default_settings(appObj,mainFrame):
+    """Adds and view default settings when Reset is pressed """
```

Comment:
We might need to consult @abhsag24, but how about we change this to 
`update_view(appObj,mainFrame,settings)`
which will extract setting from setting and apply them,
This can help us add a functionality of saving and loading different configuration files